### DISCUSSION TITLE
My course information stopped showing + I can't get the logo/icon to work properly

### DISCUSSION THREAD

**Discussion by bonni208** [LiaScript/LiaScript#159] [Q&A] [answered]
Hello. I recently had my LiaScript-generated course information stop appearing, as it was designed to do. I didn't make any edits to the text and even tried to copy/paste from the sample course to see if I was just missing something obvious and that didn't help. Is anyone able to see something in the following that would prevent the information from only showing when clicking the option (vs now, as it appears directly within the book's text/main screen)? 

![2024-01-28_08-50-02](https://github.com/LiaScript/LiaScript/assets/32577076/e50220eb-46ef-4382-bf65-9c649099e735)

Somewhat related... I've never been able to get the logo/icon working properly. The main/Home Screen of the course will show the desired graphic, but as soon as the user navigates to another page, it is the generic LiaScript logo/icon that shows. I've tried all sorts of different things to attempt to fix this (change size of image, a different format (.jpg), two different files, etc.), but nothing seems to work. 

Thank you, in advance, for any advice you have for me. I'm enjoying experimenting with LiaScript, but am very much a beginner and making all sorts of mistakes along the way. 

~ Bonni

Bonni Stachowiak
Producer + Host
Teaching in Higher Ed
Weekly episodes since 2014
[www.teachinginhighered.com](http://teachinginhighered.com/)

🗣 [How to say my name](https://www.name-coach.com/bonni-stachowiak)

**Comment by andre-dietrich** [comment, marked as answer]
Dear @bonni208 ... The main comment with all your definitions has to go first, then there is some space for custom links, icons, information that is not displayed by LiaScript, and then comes the `# Title` and your content ... Some of the definitions can be overwritten per slide, such as author and contact information or the icon. The other comments per slide have to be attached to the section-title as shown below:

``` markdown
<!--

author:   Main Author
email:    email@internet.edu
version:  0.0.1
language: en
narrator: US English Female

comment:  Business Ethics course being taught at [Vanguard University](https://teachinginhighered.com) by Dr. Bonni Stachowiak 

logo: https://innovatelearning.s3.amazonaws.com/busn435-logo.png
icon: https://innovatelearning.s3.amazonaws.com/busn435-icon.png

-->

THIS PART WILL NOT BE SHOWN BY LIASCRIPT YOU CAN USE IT TO ADD LINKS, IMAGES, ETC., SOMETHING VISIBLE ON GITHUB ...

# Business Ethics

Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.


## Second chapter
<!--
icon: https://upload.wikimedia.org/wikipedia/commons/thumb/0/0a/Antu_applications-education-university.svg/240px-Antu_applications-education-university.svg.png

author: André Dietrich
email:  LiaScript@web.de

-->

## Third chapter 

... back to the main settings
```

You can also try this out here:

[LiveEditor - Example](https://liascript.github.io/LiveEditor/?/show/code/H4sIAAAAAAAAA5VU224bNxB951dM4ZcWsHbVJnkxgiKSrdpCFSnISgmCoihG5Hh3YC7J8CI7/aN+R34sQ8k20KIP7dMuOZwzcw7P8PV3k4lSWPLg4wUAXMUG5t45hi6jHvw9452iEdnWKOzT8+6bA7q+YDQNmaIOFBN7J4emzbT5UdljsKcLIKccxoi5Fth1sHC95TTAL4JqSSntx5Fclsx5SewoJVjkgXUC7UtMBHti10PG0g8ZMMNvHx4Lw87xsW7+8vv3Q84hXbRtJmlQEtgN3A8UyTRS4QfYf/lXbqCU9b2/gKd8ds4fMJMljE5wmvSiwRH/9A7vU4Vq9yW5ly9eTWpeE1yvWFfi/x+g5h0B1GTys1Lbm2UH72bvt/BxuVrBerOF+QK6m83HNcw/wWo56y7fL99t4dNmB5eztYi5gOUWthuYXV1JfP1rdw7Lt7PrhXwX28vmHLrN24Xgrq/hw7JbzlcL2Kzherm92c2haRqlzv6pulIrH2kEDqmMYLz1EURhwJHyuVyJS6Qz5RIBDQdOul4OWZZgIiMJQFzS6A1kGoMks9Ns2BSXoWSwuBd4oHyCJhixdwho+XPBBnZZ/MKjYMPI9ecgSxzP4XPhBM6nHIsBeqCoOWMWx0GxFkftT8j1ECeulY6QHOQwEEJ1mZfmjgSkVG7gqkKK8wk4FunkxJUdRAqRBnKGohCXjYO3JUg5knaEKYhaBJqtfVJICBW4LT2LPV1tCAJGWRRx3OJBU8hUqoyigdcaScs5XQIbzDVDWITo2cgYiIpVKSmqiw1YeYO/vWXNCIYSxRodva1tYBWIRY70qGsZ5UbV2Rl0JDwN6AGlclSv65D/3aUlWI+muec7HskwNj72bV2FumqPcrnU5qGM+3baTrGduVz+wBAs66PwaSJzf/qdlOdBbNKhb396OQ0Pk/+ecBqCp0do5kz8+hdcMeXIenh+fVaMnY4c8pt72jeGHsdG6G4Hjs9sZaLF2bBHfQfZQx6qx0TPRDmLV8Xg6hvZBjrQ9QQAAA==)

**Comment by bonni208** [comment]
Dear André, 

Thank you for this incredibly-helpful response. Not only did what you have shared here immediately fix the issues I was having, but the entire OER now loads a gazillion times faster (that's a technical measurement there, of course 😇). I had no idea that chapters could have their own separate icons. It will be fun to add my graphics to each section, using that new knowledge. 

I'm grateful for the way you and your collaborators are expanding my imagination, as you increase my technological skills. Before this, I hadn't ever used GitHub in any meaningful way and had only used Markdown to write my podcast show notes. 

With gratitude and a commitment to keep on learning, 

~ B

Bonni Stachowiak
Producer + Host
Teaching in Higher Ed
Weekly episodes since 2014
[www.teachinginhighered.com](http://teachinginhighered.com/)

🗣 [How to say my name](https://www.name-coach.com/bonni-stachowiak)

**Reply by andre-dietrich** [reply]
You are welcome ... thanks for using and sharing ;-)
